{
  "uid" : "93b3de10cf81a1f2",
  "name" : "verifyGetAllAuthors",
  "fullName" : "AuthorTests.verifyGetAllAuthors",
  "historyId" : "111a852a053a1916e1b0aa3f80a210a6",
  "time" : {
    "start" : 1629113579480,
    "stop" : 1629113579533,
    "duration" : 53
  },
  "status" : "failed",
  "statusMessage" : "\nStatus code is: '405'\nResponse: '{\"timeStamp\":\"14:32:59.509 16-08-2021\",\"statusCode\":405,\"error\":\"Method Not Allowed\",\"errorMessage\":\"Method 'GET' mapped by '/api/library/author' not supported!\"}' expected [200] but found [405]",
  "statusTrace" : "java.lang.AssertionError: \nStatus code is: '405'\nResponse: '{\"timeStamp\":\"14:32:59.509 16-08-2021\",\"statusCode\":405,\"error\":\"Method Not Allowed\",\"errorMessage\":\"Method 'GET' mapped by '/api/library/author' not supported!\"}' expected [200] but found [405]\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\r\n\tat org.testng.Assert.assertEquals(Assert.java:122)\r\n\tat org.testng.Assert.assertEquals(Assert.java:907)\r\n\tat BaseTest.verifyResponse(BaseTest.java:35)\r\n\tat AuthorTests.verifyGetAllAuthors(AuthorTests.java:31)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.util.ArrayList.forEach(ArrayList.java:1257)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)\r\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)\r\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)\r\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)\r\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "\nStatus code is: '405'\nResponse: '{\"timeStamp\":\"14:32:59.509 16-08-2021\",\"statusCode\":405,\"error\":\"Method Not Allowed\",\"errorMessage\":\"Method 'GET' mapped by '/api/library/author' not supported!\"}' expected [200] but found [405]",
    "statusTrace" : "java.lang.AssertionError: \nStatus code is: '405'\nResponse: '{\"timeStamp\":\"14:32:59.509 16-08-2021\",\"statusCode\":405,\"error\":\"Method Not Allowed\",\"errorMessage\":\"Method 'GET' mapped by '/api/library/author' not supported!\"}' expected [200] but found [405]\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\r\n\tat org.testng.Assert.assertEquals(Assert.java:122)\r\n\tat org.testng.Assert.assertEquals(Assert.java:907)\r\n\tat BaseTest.verifyResponse(BaseTest.java:35)\r\n\tat AuthorTests.verifyGetAllAuthors(AuthorTests.java:31)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.util.ArrayList.forEach(ArrayList.java:1257)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)\r\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)\r\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)\r\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)\r\n",
    "steps" : [ {
      "name" : "getAllEntities",
      "time" : {
        "start" : 1629113579481,
        "stop" : 1629113579511,
        "duration" : 30
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "entity",
        "value" : "author"
      } ],
      "hasContent" : true,
      "stepsCount" : 0,
      "shouldDisplayMessage" : false,
      "attachmentsCount" : 0
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : true,
    "stepsCount" : 1,
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 0
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "package",
    "value" : "AuthorTests"
  }, {
    "name" : "testClass",
    "value" : "AuthorTests"
  }, {
    "name" : "testMethod",
    "value" : "verifyGetAllAuthors"
  }, {
    "name" : "parentSuite",
    "value" : "Surefire suite"
  }, {
    "name" : "suite",
    "value" : "Surefire test"
  }, {
    "name" : "subSuite",
    "value" : "AuthorTests"
  }, {
    "name" : "host",
    "value" : "Kompukter"
  }, {
    "name" : "thread",
    "value" : "5308@Kompukter.main(1)"
  }, {
    "name" : "framework",
    "value" : "testng"
  }, {
    "name" : "language",
    "value" : "java"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "a78914440bbe9f9f",
      "status" : "passed",
      "time" : {
        "start" : 1629110373617,
        "stop" : 1629110373649,
        "duration" : 32
      }
    }, {
      "uid" : "c2df0d9a2e746105",
      "status" : "passed",
      "time" : {
        "start" : 1629109759200,
        "stop" : 1629109759227,
        "duration" : 27
      }
    }, {
      "uid" : "83c6063a5817bb24",
      "status" : "passed",
      "time" : {
        "start" : 1629107458405,
        "stop" : 1629107458460,
        "duration" : 55
      }
    }, {
      "uid" : "513203f4e5c2f0fb",
      "status" : "broken",
      "statusDetails" : "Connection refused: connect",
      "time" : {
        "start" : 1629107299828,
        "stop" : 1629107303914,
        "duration" : 4086
      }
    }, {
      "uid" : "8ddc7befb21080db",
      "status" : "passed",
      "time" : {
        "start" : 1629105310118,
        "stop" : 1629105310314,
        "duration" : 196
      }
    }, {
      "uid" : "7521cd0616cd1b6a",
      "status" : "passed",
      "time" : {
        "start" : 1629105026030,
        "stop" : 1629105026112,
        "duration" : 82
      }
    }, {
      "uid" : "adf7c9e1dba980da",
      "status" : "passed",
      "time" : {
        "start" : 1629104997499,
        "stop" : 1629104997528,
        "duration" : 29
      }
    }, {
      "uid" : "7ed8064662448f90",
      "status" : "passed",
      "time" : {
        "start" : 1629104548277,
        "stop" : 1629104548313,
        "duration" : 36
      }
    }, {
      "uid" : "d6be55cc68480ffb",
      "status" : "passed",
      "time" : {
        "start" : 1629059788855,
        "stop" : 1629059789132,
        "duration" : 277
      }
    }, {
      "uid" : "689c5e2bb2ad2b96",
      "status" : "passed",
      "time" : {
        "start" : 1629059333097,
        "stop" : 1629059333127,
        "duration" : 30
      }
    }, {
      "uid" : "464f86ce49d6e803",
      "status" : "passed",
      "time" : {
        "start" : 1629059190751,
        "stop" : 1629059190805,
        "duration" : 54
      }
    }, {
      "uid" : "e73b61972a6cf15e",
      "status" : "passed",
      "time" : {
        "start" : 1629058752175,
        "stop" : 1629058752217,
        "duration" : 42
      }
    }, {
      "uid" : "f9877cd63d3e6c51",
      "status" : "passed",
      "time" : {
        "start" : 1629058505295,
        "stop" : 1629058505324,
        "duration" : 29
      }
    }, {
      "uid" : "78d4d0d95e060cdd",
      "status" : "passed",
      "time" : {
        "start" : 1629058371608,
        "stop" : 1629058371641,
        "duration" : 33
      }
    }, {
      "uid" : "a4443b5726d45d8",
      "status" : "passed",
      "time" : {
        "start" : 1629057048102,
        "stop" : 1629057048147,
        "duration" : 45
      }
    }, {
      "uid" : "f8d88e1ad3c8105e",
      "status" : "passed",
      "time" : {
        "start" : 1629056727933,
        "stop" : 1629056727982,
        "duration" : 49
      }
    } ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ "failed" ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "93b3de10cf81a1f2.json",
  "parameterValues" : [ ]
}